app/templates/common/search-view.pug

```diff
@@ -9,6 +9,9 @@ block content
         li.active(data-i18n=currentEditor)
           | #{currentEditor}
 
+    div #{document.location.href.split('/').pop().toUpperCase()}
+    hr
+
```

Comment:
What is this for? Don't think we need it.
<img width="957" alt="Screen Shot 2022-06-15 at 17 13 04" src="https://user-images.githubusercontent.com/99704/173962744-84bdfd14-5fcd-42c1-b4a7-87a602b3c7eb.png">